chapter1/Example_1_2_Bouncing_Ball_Vectors/Example_1_2_Bouncing_Ball_Vectors.pde

```diff
@@ -0,0 +1,34 @@
+// The Nature of Code
+// Daniel Shiffman
+// http://natureofcode.com
+
+// Example 1-2: Bouncing Ball, with PVector!
+//{!2 .bold} Instead of a bunch of floats, we now just have two variables.
+PVector position;
+PVector velocity;
+
+void setup() {
+  size(640, 360);
+  //{!2 .bold} Note how PVector() has to be called inside of setup().
+  position = new PVector(100, 100);
+  velocity = new PVector(2.5, 2);
+}
+
+void draw() {
+  background(255);
+  //{!1 .bold .no-comment}
+  position.add(velocity);
+
+  //{!6 .bold .code-wide} We still sometimes need to refer to the individual components of a PVector and can do so using the dot syntax: position.x, velocity.y, etc.
```

Comment:
```suggestion
  // We still sometimes need to refer to the individual components of a PVector and can do so using the dot syntax: position.x, velocity.y, etc.
```

This is some leftover markup for the book publication formatting that is not necessary in the code examples!